pytorch_fid/fid_score.py

```diff
@@ -209,35 +209,45 @@ def calculate_activation_statistics(files, model, batch_size=50, dims=2048, devi
     -- sigma : The covariance matrix of the activations of the pool_3 layer of
                the inception model.
     """
-    act = get_activations(files, model, batch_size, dims, device)
+    act = get_activations(files, model, batch_size, dims, device,transforms)
     mu = np.mean(act, axis=0)
     sigma = np.cov(act, rowvar=False)
     return mu, sigma
 
 
-def _compute_statistics_of_path(path, model, batch_size, dims, device):
+def _compute_statistics_of_path(path, model, batch_size, dims, device,transforms=TF.ToTensor()):
     if path.endswith('.npz'):
         f = np.load(path)
         m, s = f['mu'][:], f['sigma'][:]
         f.close()
+    elif os.path.isfile(path+'/paths.txt'):
```

Comment:
Maybe you can pass "paths.txt" directly

Reply:
This was done intentionally, the user only has to give the path to the repo wich contains either images or a paths.txt.

Reply:
ok